src/controllers/userController.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable camelcase */
+/* eslint-disable valid-jsdoc */
+import Helper from '../helpers/helperUtils';
+import userService from '../services/userService';
+/**
+ * @class UsersController
+ */
+class UserController {
+  /** Login User
+   * @static
+   * @params {*} req
+   * @params {*} res
+   * @returns {object} loginUsers
+   */
+  static async loginAUser(req, res) {
+    const { email } = req.body;
+    try {
+      const loggedUser = await userService.loginAUser(email);
+      if (loggedUser) {
+        const validatePassword = Helper.verifyPassword(loggedUser.password, req.body.password);
+        if (validatePassword) {
+          const token = Helper.generateToken(loggedUser.dataValues);
```

Comment:
Please be more precise on the data in your token

Reply:
OK, done.